src/js/components/conversation.jsx

```diff
@@ -22,6 +29,11 @@ export class ConversationComponent extends Component {
         errorNotificationMessage: PropTypes.string
     };
 
+    constructor(...args) {
```

Comment:
don't need the constructor to do this, you can use a class property... just do  `debounceOnScroll = debounce(this.onScroll.bind(this), 200);`.... like `scrollTimeouts` just below